themes/default/data/registry/packages/aws-static-website.yaml

```diff
@@ -0,0 +1,14 @@
+category: Cloud
+component: true
+description: ""
```

Comment:
Should we have a description here? Looks like other packages have one.

Reply:
just added one. thanks. I'll get those license fields updated in the provider repo.